RCS file: /cvs/gnome/ooo-build configure.in,v
retrieving revision 1.332
retrieving revision 1.333
diff -u -r1.332 -r1.333
--- configure.in	2005/12/20 00:08:41	1.332
+++ configure.in	2005/12/22 12:20:31	1.333
@@ -164,12 +164,12 @@
 
 AC_ARG_ENABLE(gtk,
 [
-  --disable-gtk           Disables gtk+ native widgets and Gnome icons.],
+  --disable-gtk           Disables gtk+ native widgets.],
 ,)
 
 AC_ARG_ENABLE(kde,
 [
-  --disable-kde           Disables KDE native widgets and KDE icons.],
+  --disable-kde           Disables KDE native widgets.],
 ,)
 
 AC_ARG_ENABLE(gcc33,
@@ -475,22 +475,16 @@
 
 AC_MSG_CHECKING([for widget sets])
 OOO_WIDGET_FLAGS=
-WITH_ICONS="hicontrast"
-if test "z$with_win32" != "z"; then
-   WITH_ICONS="industrial $WITH_ICONS"
-fi
 if test "$enable_kde" != "no"; then
    if test "z$with_win32" = "z"; then
       OOO_WIDGET_FLAGS="--enable-kde "
       widget_sets="kde"
-      WITH_ICONS="crystal $WITH_ICONS"
    fi
 fi
 if test "$enable_gtk" != "no"; then
    if test "z$with_win32" = "z"; then
       OOO_WIDGET_FLAGS="--enable-gtk $OOO_WIDGET_FLAGS"
       widget_sets="gtk $widget_sets"
-      WITH_ICONS="industrial $WITH_ICONS"
 
       PKG_CHECK_MODULES( FOO_GTK, [ gtk+-2.0 ],
          have_gtk=true, have_gtk=false )
@@ -510,7 +504,6 @@
 
 AC_MSG_RESULT($OOO_WIDGET_FLAGS)
 AC_SUBST(OOO_WIDGET_FLAGS)
-AC_SUBST(WITH_ICONS)
 
 if test "z$BUILD_WIN32" = "zyes" -a "z$enable_cairo" = "z" ; then
     AC_MSG_WARN([On Cygwin, automatically using --disable-cairo])
@@ -804,7 +797,6 @@
 	src package dir:    $SRCDIR
 	distro:             $with_distro
 	widget sets:        ${widget_sets:-disabled}
-	icons:              ${WITH_ICONS:-default}
 	gcc to use:         $gcc_to_use
 	hunspell-uno:	    $hunspell_uno_enabled
 	hunspell-lib:	    $hunspell_lib_result
===================================================================
RCS file: /cvs/gnome/ooo-build/bin setup.in,v
retrieving revision 1.153
retrieving revision 1.154
diff -u -r1.153 -r1.154
--- bin/setup.in	2005/12/02 19:58:51	1.153
+++ bin/setup.in	2005/12/22 12:20:32	1.154
@@ -18,7 +18,6 @@
 ICECREAM_JOBS='@ICECREAM_JOBS@'
 ICECREAM_BINDIR='@ICECREAM_BINDIR@'
 OOO_WIDGET_FLAGS='@OOO_WIDGET_FLAGS@'
-WITH_ICONS='@WITH_ICONS@'
 OOO_CUSTOM_IMAGES='@OOO_CUSTOM_IMAGES@'
 OOO_CRYSTAL_IMAGES='@OOO_CRYSTAL_IMAGES@'
 OOO_BUILDVERSION='@OOO_BUILDVERSION@'
@@ -73,10 +72,6 @@
 # build the right native widgets
 CONFIGURE_OPTIONS="$CONFIGURE_OPTIONS $OOO_WIDGET_FLAGS"
 
-if test "z$WITH_ICONS" != "z"; then
-    CONFIGURE_OPTIONS="$CONFIGURE_OPTIONS --with-images='$WITH_ICONS'"
-fi
-
 # Misc. internal
 TOOLSDIR='@TOOLSDIR@'
 OOBUILDDIR='@OOBUILDDIR@'
===================================================================
RCS file: /cvs/gnome/ooo-build/bin/unpack,v
retrieving revision 1.62
retrieving revision 1.63
diff -u -r1.62 -r1.63
--- bin/unpack	2005/11/30 13:07:36	1.62
+++ bin/unpack	2005/12/22 12:20:32	1.63
@@ -254,7 +254,7 @@
 # fix a checkbox bug - in the up-stream icon set.
     rm -f $CUSTOM_DIR/industrial/vcl/source/src/check*; 
 fi
-if test "z${WITH_ICONS#*crystal}" != "z$WITH_ICONS" ; then
+if test -f $SRCDIR/$OOO_CRYSTAL_IMAGES ; then
     tar xjf $SRCDIR/$OOO_CRYSTAL_IMAGES -C $CUSTOM_DIR || exit 1;
 fi
 
